content/_layouts/advisory.html.haml

```diff
@@ -4,7 +4,7 @@ section: security
 ---
 
 :ruby
-  plugins = page.issues.collect { |issue| issue.plugins }.flatten.uniq.sort { |x,y| site._generated[:update_center].plugins[x.name].title <=> site._generated[:update_center].plugins[y.name].title }
+  plugins = page.issues.collect { |issue| issue.plugins }.flatten.uniq.keep_if { |p| p }.sort { |x,y| site._generated[:update_center].plugins[x.name].title <=> site._generated[:update_center].plugins[y.name].title }
```

Comment:
Null safety (the 2017-11-08 advisory has issue entries without associated plugins)